tools/release/bundleWinFiles.sh.cmake

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
hmm can you fix the rename? it seems git didnt caught that correctly (`git mv a b`)

Reply:
Maybe not in the full PR but the commit has it:
```
git show 92bfc140090764e15164a995f17ec87a4ab59aed
diff --git a/tools/release/prepareRelease.sh.cmake b/tools/release/bundleWinFiles.sh.cmake
similarity index 98%
rename from tools/release/prepareRelease.sh.cmake
rename to tools/release/bundleWinFiles.sh.cmake
```

Reply:
hmm interesting.